integration/strongswan/server/ecdsa/serverKey.pem

```diff
@@ -0,0 +1,6 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
Seeing private keys, passwords, API keys, etc put into a public github repo always makes me twitch.  I _know_ that this is a horrible practice, and I keep having to convince myself that it's ok for integration test / poc...